src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -282,12 +285,15 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
         );
         const nonSegmentAttributes = getAttributeItemsWithoutStacks(buckets);
         if (nonSegmentAttributes.length > 1 && isEmpty(segments)) {
-            // first attribute is taken, find next available non-date attribute
-            const [, ...attributesWithoutFirst] = nonSegmentAttributes;
-            const nonDate = attributesWithoutFirst.filter(
-                (attribute: IBucketItem) => !isDateBucketItem(attribute),
-            );
-            segments = nonDate.slice(0, 1);
+            segments = nonSegmentAttributes
+                .filter(
+                    (item: IBucketItem): boolean => {
```

Comment:
For consistency, please replace `item` with `attribute`.
Should we merge these 2 filter calls into 1?

Reply:
Thanks. Will change item -> attribute.
But i would like to keep 2 filter separately because of better readable

Reply:
updated